app/src/main/res/layout/article_view.xml

```diff
@@ -0,0 +1,36 @@
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+
+<LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal">
+    <TextView
+        android:id="@+id/authorLabel"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/author"/>
```

Comment:
This is a good idea in order to keep the labels separate from the actual associated values. It doesn't look like they're referenced in the `ArticleView` class though.  Not a requirement for week 3 but it may be good to just create string resources for authorLabel and titleLabel and set them in the XML directly with the `android:text` attribute.